src/NuGet.Core/NuGet.Packaging/Licenses/LicenseExpressionTokenizer.cs

```diff
@@ -34,7 +34,7 @@ internal LicenseExpressionTokenizer(string value)
         /// <returns>Whether the value has valid characters.</returns>
         internal bool HasValidCharacters()
         {
-            var regex = new Regex("^[a-zA-Z0-9\\.\\-\\s\\+\\(\\)]+$");
+            var regex = new Regex("^[a-zA-Z0-9\\.\\-\\s\\+\\(\\)]+$", RegexOptions.CultureInvariant);
```

Comment:
Below, there are tests for this method:

https://github.com/NuGet/NuGet.Client/blob/d39e857e6b1265d1b4990cbe660e2a0ad259e0cf/test/NuGet.Core.Tests/NuGet.Packaging.Test/LicensesTests/LicenseExpressionTokenizerTests.cs#L124-L129